kornia/augmentation/random_generator/_2d/cutmix.py

```diff
@@ -23,15 +24,15 @@ class CutmixGenerator(RandomGeneratorBase):
     Args:
         p (float): probability of applying cutmix.
         num_mix (int): number of images to mix with. Default is 1.
-        beta (torch.Tensor, optional): hyperparameter for generating cut size from beta distribution.
+        beta (Tensor, optional): hyperparameter for generating cut size from beta distribution.
```

Comment:
```suggestion
        beta: hyperparameter for generating cut size from beta distribution.
```
parameters are automatically generated by sphinx. However, not sure this classes are added to the docs